### PR TITLE
Env markers

### PR DESCRIPTION
**Pull request by richafrank**
Use environment markers instead of a separate requirements file for py2.

### CODE REVIEW

**Comment by richafrank** [comment]
@llllllllll What do you think of this replacement for requirements_py2?

File: setup.py [hunk 1]
```diff
@@ -160,30 +157,40 @@ def _with_bounds(req):
...
+            return 'numpy x.x'
+
+        formatted = '%s %s%s' % ((name,) + m.group('comp', 'spec'))
+        pycomp, pyspec = m.group('pycomp', 'pyspec')
+        if pyspec:
+            comp = STR_TO_CMP[pycomp]
+            pyver_spec = StrictVersion(pyspec)
+            selector = []
+
+            for pyver in ('2.7', '3.4'):
```
Comment (llllllllll) [inline setup.py]: I think we could just do something like: `selector = 'int(py) %s %s' % (pycomp, pyver_spec)`. Conda selectors have the name `py` available as a 2 character string like `'34'`.

**Comment by llllllllll** [comment]
just had one comment about the translation to a conda selector, but otherwise lgtm. I wonder if we should make a package like `versioneer` that has our cython stuff and now our conda translation. I feel like this is a problem other people also have.

File: setup.py (hunk 1 above)
Comment (richafrank) [inline setup.py]: Oh good point - the selector is just a string of python!

**Pull request merged**